practice-two/src/javascript/main.js

```diff
@@ -1,4 +1,8 @@
 /* global document */
+
 import { renderAllNotes } from './modules/getNotes';
 
 document.addEventListener('DOMContentLoaded', renderAllNotes);
+
+import { renderNotesByCategory } from './modules/filterNotes';
```

Comment:
@quanghuy1799 Let's move importing to the top

Reply:
yes, i will